Assets/Shopify/UIToolkit/Shops/Generic/Classes/MultiProductListItem.cs

```diff
@@ -10,15 +10,17 @@ namespace Shopify.UIToolkit.Shops {
 
     public class MultiProductListItem : MonoBehaviour {
         public RemoteImageLoader ImageLoader;
+        public UnityEngine.UI.Image Image;
+        public Sprite ErrorImage;
         public Text TitleLabel;
         public Text PriceLabel;
         public Text DescriptionLabel;
-
+        public AspectRatioFitter AspectRatioFitter;
```

Comment:
Need the fitter so that the product images can fill the space they are allocated